errors.toml

```diff
@@ -941,6 +941,11 @@ error = '''
 regions not completely left cover span, span %v regions: %v
 '''
 
+["CDC:ErrRenameTablesTableNotFound"]
+error = '''
+failed to find table id(s): '%v' in cdc schema snapshot, ddl query: [%s], if you want to replicate these table(s), please add them to filter rule.
```

Comment:
I think this error message is not precise. 
We fail for case 8 because the atomicity of `rename tables`,  but not the `can't find table schema`. Or we can't explain why  DDL `rename t5 to t55` will not raise an error.